### ISSUE TITLE
Attribution help content needs typographic love

### ISSUE THREAD

**Issue by edtjones** [layersoflondon/application#50]
The help content on the 'using someone else's stuff?' page needs some love. Long line lengths and tight line spacing make it hard to parse.

The CTA needs to be a button or something - 'yes, I need to credit someone'

![screen shot 2016-12-08 at 21 58 48](https://cloud.githubusercontent.com/assets/4377783/21029434/9a023da8-bd91-11e6-8cea-5559d3748263.png)

**Comment by andyError** [comment]
Done.
